cli/internal/run/dry_run.go

```diff
@@ -142,18 +143,19 @@ func executeDryRun(ctx gocontext.Context, engine *core.Engine, g *graph.Complete
 		}
 
 		taskIDs = append(taskIDs, taskSummary{
-			TaskID:          packageTask.TaskID,
-			Task:            packageTask.Task,
-			Package:         packageTask.PackageName,
-			Hash:            hash,
-			CacheState:      itemStatus,
-			Command:         command,
-			Dir:             packageTask.Pkg.Dir.ToString(),
-			Outputs:         packageTask.TaskDefinition.Outputs.Inclusions,
-			ExcludedOutputs: packageTask.TaskDefinition.Outputs.Exclusions,
-			LogFile:         packageTask.RepoRelativeLogFile(),
-			Dependencies:    stringAncestors,
-			Dependents:      stringDescendents,
+			TaskID:                 packageTask.TaskID,
+			Task:                   packageTask.Task,
+			Package:                packageTask.PackageName,
+			Hash:                   hash,
+			CacheState:             itemStatus,
+			Command:                command,
+			Dir:                    packageTask.Pkg.Dir.ToString(),
+			Outputs:                packageTask.TaskDefinition.Outputs.Inclusions, // TODO: remove this?
+			ExcludedOutputs:        packageTask.TaskDefinition.Outputs.Exclusions, // TODO: remove this?
```

Comment:
Maybe add some more context around why these are redundant and can be removed in the future if we're gonna leave the `TODO` here.